packages/fxa-content-server/app/scripts/views/pair/index.js

```diff
@@ -25,6 +25,7 @@ class PairIndexView extends FormView {
     ...FormView.prototype.events,
     'click #get-fx-mobile': 'downloadLinkEngagement',
     'click #pair-not-now': 'pairNotNowHandler',
```

Comment:
Does this need to be removed since the id attr was changed?

Reply:
This id was used for two elements in the mustache, I've changed one of the two instances only so they could emit different metrics. The original #pair-not-now is related to `FXA-9593` (@dschom fyi, this second one is assigned to you)